scripts/sv/delete_cluster.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# This script creates a Google Dataproc cluster suitable for running the GATK-SV pipeline.
+
+set -eu
+if [ "$#" -ne 1 ]; then
+    echo "Please provide name of cluster to be shutdown"
+    exit 1
+fi
+
+CLUSTER_NAME=$1
+
+gcloud dataproc clusters delete "$CLUSTER_NAME" --async
```

Comment:
Doesn't seem like it according to [this](https://cloud.google.com/dataproc/docs/guides/manage-cluster#delete_a_cluster)